cognite/powerops/resync/models/v2/dms/data_models/views/asset-Watercourse.view.yaml

```
@@ -30,8 +38,8 @@ properties:
     description: The production obligation for the Watercourse.
   plants:
     type:
-      space: '{{type_space}}'
-      externalId: isSubAssetOf
+      space: '{{asset_space}}'
+      externalId: Watercourse.plants
```

In the asset model, lets keep these as is it gives more flexibility on filtering. 
```suggestion
      space: '{{type_space}}'
      externalId: isSubAssetOf
```